products/dependencytrack.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Dependency-Track
+category: server-app
+permalink: /dependencytrack
+alternate_urls:
+-   /dependency-track
+eolColumn: Security Support
+eolWarnThreshold: 121
+releaseColumn: Latest
+auto:
+  - git: https://github.com/apache/maven.git
+    regex: ^v(?<major>0|[1-9]\d*)_(?<minor>0|[1-9]\d*)_(?<patch>\d{1,3})_?(?<tiny>\d+)?$
+    template: '{{major}}.{{minor}}.{{patch}}{%if tiny %}p{{tiny}}{%endif%}'
+  - dockerhub: dependencytrack/bundled
```

Comment:
Docker tags are often re-pushed, so we prefer using source control tags, and docker images only as a last resort.

Reply:
I removed the docker reference